.github/workflows/velox_docker.yml

```diff
@@ -51,7 +51,7 @@ concurrency:
 jobs:
   build-native-lib-centos-7:
     runs-on: ubuntu-20.04
-    container: apache/gluten:gluten-vcpkg-builder_2024_08_05 # centos7 with dependencies installed
+    container: apache/gluten:vcpkg-centos-7
```

Comment:
@zhztheplayer, yes, it would be better to also do this for centos-8 build. At least, those libs generated by arrow build (cpp/java) can be cached in docker to accelerate CI job when the current folder based cache miss happens.

Reply:
can we reuse the binary on centos 7?

Reply:
@zhouyuan, I will try this.